tests/integration/test_script_runner.py

```diff
@@ -46,6 +47,31 @@ class BackendDependencies(enum.Enum):
         "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
     },
     # {
+    #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/inferred_and_runtime_yaml_example.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    # },
+    # {
+    #     "user_flow_script": "tests/integration/docusaurus/connecting_to_your_data/cloud/azure/pandas/inferred_and_runtime_python_example.py",
+    #     "data_context_dir": "tests/integration/fixtures/no_datasources/great_expectations",
+    # },
+    # GCS
```

Comment:
Segmenting by cloud provider and writing out the fixtures for the remaining Azure tests. If we like this, we could apply these changes to the S3 tests (once we've fixed them up!)